float64[] acc_gains
float64[] i_gains
float64[] i_claim
---

